### ISSUE TITLE
Can't find 'file-loader' issue with react-scripts 5.0.0

### ISSUE THREAD

**Issue by anglee** [DocSpring/craco-less#86]
react-scripts version 5.0.0
@craco/craco version 6.4.3
craco-less version 1.20.0

```
/foobar/my-app/node_modules/@craco/craco/lib/plugin-utils.js:29
    throw new Error(
          ^

Error: Can't find file-loader in the development webpack config!

```

**Comment by saifulshihab** [comment]
same here :(

**Comment by Hjj1501600921** [comment]
why

**Comment by xyy94813** [comment]
CRA v5 no longer uses file-loader.

this line cause the problem:

https://github.com/DocSpring/craco-less/blob/a4c93b1b3ab44bedbfed07a676641c4d7635c72d/lib/craco-less.js#L163

**Comment by dr2009** [comment]
#87

**Comment by dr2009** [comment]
fixed it #87 waiting pr received

if you want use it immediately. try `craco-less-fix`. i just publish for temporary use

**Comment by MalazAlkoj** [comment]
https://github.com/navikt/sykefravarsstatistikk/pull/336

**Comment by MyNameIsAlaa** [comment]
any update please?

**Comment by Shasha-beep** [comment]
```
    throw new Error(
          ^

Error: Can't find file-loader in the development webpack config!

This error probably occurred because you updated react-scripts or craco. Please try updating craco-less to 
the latest version:

   $ yarn upgrade craco-less

Or:

   $ npm update craco-less
```
What's the problem

**Comment by chentle** [comment]
> why

**Comment by chentle** [comment]
yarn add craco-less-fix, and remove the craco-less,rename the craco-less-fix as craco-less,then it's done,remember to change the data in the craco.config.js

**Comment by chentle** [comment]
> fixed it #87 waiting pr received
> 
> if you want use it immediately. try `craco-less-fix`. i just publish for temporary use

good,thank u

**Comment by flyingCrp** [comment]
is there any progress ? i planning to upgrade my project. 😄
